### PR TITLE
Pin redis

### PR DESCRIPTION
**Pull request by fao89**
https://github.com/pulp/pulp_installer/runs/5661786624?check_suite_focus=true

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ pygtrie~=2.4.2
 psycopg2~=2.9.1
 PyYAML>=5.1.1,<5.5.0
 python-gnupg~=0.4.7
-redis>=3.4.0
+redis>=3.4.0,<4.2.0
 rq~=1.9.0
```
Comment (mdellweg) [inline requirements.txt:29]: Did we forget to remove rq?

**Review by mdellweg — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (fao89) [reply requirements.txt:29]: this is 3.15 branch

Comment (mdellweg) [reply requirements.txt:29]: I see. But shouldn't we create the fix on the newest branch we want it and backport from there?

Comment (fao89) [reply requirements.txt:29]: yes, I will re-do the PR

**Pull request closed without merging**